Makefile

```diff
@@ -0,0 +1,75 @@
+#Python3 and virtual environment
+PY3 := $(shell command -v python3 2> /dev/null)
+VENV := .venv
+PYTHON3 := $(VENV)/bin/python3
+
+#If caramel_tool exists in the venv caramel has been installed
+INSTALLED := $(VENV)/bin/caramel_tool
+
+#Terminal formatting
+BOLD := tput bold
+PASS := $(BOLD); tput setaf 2
+FAIL := $(BOLD); tput setaf 1
+LINE := echo "---------------------------------------"
+RESET_TERM := tput op; tput sgr0
+BLR := $(BOLD); $(LINE); $(RESET_TERM) #Bold Line, Reset formatting
+
+#Check for python3 install and virtual environment
+$(PYTHON3):
+	@if [ -z $(PY3) ]; then \
+		$(FAIL);\
+		echo "Python 3 could not be found.";\
+		$(RESET_TERM);\
+		exit 2; \
+	fi
+	$(PY3) -m venv $(VENV)
+
+
+#Install the project via setup.py
+.PHONEY: install
+install: $(INSTALLED)
```

Comment:
This rule doesn't actually "make install"  as end-users expect "make install" to work from an traditional way.

prefix it with venv- instead, or remove the "install" alias completely.